core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -168,6 +171,16 @@ public DefaultRequestLog(RequestContext ctx, ContentPreviewerFactory requestCont
                                                               "responseContentPreviewerFactory");
     }
 
+    @Override
+    public <T> Attribute<T> attr(AttributeKey<T> key) {
+        return ctx.attr(key);
+    }
+
+    @Override
+    public <T> boolean hasAttr(AttributeKey<T> key) {
+        return ctx.hasAttr(key);
+    }
+
```

Comment:
Could we also add `attrs()` method like `RequestContext` has?

Reply:
Added. Thanks!